api/server.js

```diff
@@ -82,8 +82,9 @@ process
     serverUtils.serverError(err, req, res);
   });
 
-  app.listen(apiPort, () => {
+  const server = app.listen(apiPort, () => {
     logger.info('Medic API listening on port ' + apiPort);
   });
-
+  const ONE_HOUR = 60 * 60 * 1000;
```

Comment:
Why did you pick one hour? If Node v14 has no timeout, why not use infinity (or max int)? This would allow the actual timeout to be set in the infrastructure rather than hardcoded in API.

Also, rename the const to describe what it's used for, not the value it holds, eg...

```suggestion
  const SOCKET_TIMEOUT = 60 * 60 * 1000; // one hour
```

Finally I think it's cleaner to have all constants together at the top of the file. It makes it easier to find variables to reuse. More generally it's good practice because execution errors are found when the module is included as opposed to delayed at some point during execution (though this doesn't apply to this specific example).

Reply:
The reason for picking one hour was in the PR description, it's the same timeout we use for the AWS load balancer.

Reply:
> why not use infinity (or max int)

Using `0` as the value means there is no timeout. 
The reason for setting it to one hour is the same reason I suggested we dial down the `fabric.request_timeout` in our infrastructure: so we reduce the chance of one (a few) very large requests from effectively ddos-ing us. 
API doesn't do much processing right now, on any endpoint, so DDOS at API level is unlikely, though.